### PR TITLE
RHCLOUD-13501: Create config manager connector

### PR DESCRIPTION
**Pull request by aleccohan**
Created config manager connector for upcoming check for "remediations: enabled" state.

JIRA:  https://issues.redhat.com/browse/RHCLOUD-13501
    
    ## Secure Coding Practices Checklist GitHub Link
    - https://github.com/RedHatInsights/secure-coding-checklist

    ## Secure Coding Checklist
    - [ ] Input Validation
    - [ ] Output Encoding
    - [ ] Authentication and Password Management
    - [ ] Session Management
    - [ ] Access Control
    - [ ] Cryptographic Practices
    - [ ] Error Handling and Logging
    - [ ] Data Protection
    - [ ] Communication Security
    - [ ] System Configuration
    - [ ] Database Security
    - [ ] File Management
    - [ ] Memory Management
    - [ ] General Coding Practices

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -100,6 +100,14 @@ const config = {
         revalidationInterval: parseIntEnv('COMPLIANCE_REVALIDATION_INVERVAL', 60 * 60) // 1 hour
     },
 
+    configManager: {
+        impl: env.CONFIG_MANAGER_IMPL,
+        host: env.CONFIG_MANAGER_HOST || 'http://config-manager-api.config-manager-ci.svc.cluster.local:8000',
```
Comment (Jason-RH) [inline src/config/index.js:105]: port 8081 in CI

File: src/connectors/configManager/impl.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        uri.segment('current');
+
+        const options = {
+            uri: uri.toString(),
+            method: 'GET',
+            json: true,
+            headers: this.getForwardedHeaders()
+        };
+
+        const result = await this.doHttp (options, false, this.fetchRunHosts);
```
Comment (Jason-RH) [inline src/connectors/configManager/impl.js:29]: Should this be passing the result to fetchRunHosts? (I haven't looked into how this works past the PR)

File: src/config/index.js (hunk 1 above)
Comment (aleccohan) [reply src/config/index.js:105]: @Jason-RH does the rest of the hostname look ok for talking too config-manager?

File: src/connectors/configManager/impl.js (hunk 2 above)
Comment (aleccohan) [reply src/connectors/configManager/impl.js:29]: nope!  Just fixed it!  :)

File: src/config/index.js (hunk 1 above)
Comment (Jason-RH) [reply src/config/index.js]: @aleccohan Looks like it's actually config-manager-service instead of config-manager-api, but looks good otherwise

Comment (Jason-RH) [reply src/config/index.js]: I may update that later to be more consistent. I'll put up a PR against remediations when I do

Comment (aleccohan) [reply src/config/index.js]: sounds great!  just pushed up those changes

**Review by Jason-RH — APPROVED** [review]

**Pull request merged**